packages/marketplace/src/resources/locales/fr.json

```diff
@@ -66,6 +66,10 @@
       "loading": "Chargement…",
       "errorLoading": "Erreur lors du chargement du README"
     },
+    "backup-restore": {
+      "col description": "Sauvegarder et restaurer toutes les données du Marché. Celà n’inclue pas les réglages ou toutes autres choses ayant été installées via le Marché."
+      "col action": "Sauvegarder / Restaurer"
+    },
```

Comment:
Please use `en.json` as a template and only change the values.

If the keys is not present there then it is not used.